sparse/layout_test.go

```diff
@@ -1,192 +0,0 @@
-package sparse
```

Comment:
we are not using RetrieveLayout() function in our new client/server. But let me try to put it back in if it compiles. We can use its test cases to generate our own tests using new functions maybe later.